src/fixate/drivers/dmm/keithley_6500.py

```diff
@@ -0,0 +1,302 @@
+from threading import Lock
+from pyvisa import constants
+from fixate.core.exceptions import InstrumentError, ParameterError
+from fixate.core.common import mode_builder, deprecated
+from fixate.drivers.dmm.helper import DMM
+import time
+
+
+class Keithley6500(DMM):
+    REGEX_ID = "KEITHLEY INSTRUMENTS,MODEL DMM6500"
+    INSTR_TYPE = "VISA"
+
+    def __init__(self, instrument, *args, **kwargs):
+        self.instrument = instrument
+        instrument.rtscts = 1
+        self.lock = Lock()
+        self._display = "ON100"
+        # del self.instrument.timeout
+        self.instrument.timeout = 10000
+        self.instrument.query_delay = 0.3  # Stop DMM crash
+        self.instrument.delay = 0  # Stop DMM crash
+        self.is_connected = True
+        self.reset()
+        self._samples = 1
+        self._CLEAN_UP_FLAG = False
+        self._ANALOG_FLAG = False
+        self._DIGITAL_FLAG = False
+        self._range_string = ""
+        self._bandwidth = None
+        self._mode = None
+        # Set to True to have explicit error checks on each read
+        self.legacy_mode = False
+        self._modes = {
+            "voltage_ac": "VOLT:AC",
+            "voltage_dc": "VOLT:DC",
+            "current_ac": "CURR:AC",
+            "current_dc": "CURR:DC",
+            "resistance": "RES",
+            "fresistance": "FRES",
+            "period": "PER",
+            "frequency": "FREQ",
+            "temperature": "TEMP",
+            "capacitance": "CAP",
+            "continuity": "CONT",
+            "diode": "DIOD",
+        }
+
+        self._init_string = ""  # Unchanging
+
+    # Adapter for different DMM behaviour
+    @property
+    def display(self):
+        return self.display
+
+    @display.setter
+    def display(self, val):
+        if val == "off":
+            self._display = "OFF"
+        else:
+            self._display = "ON100"
+
+        self._write(["DISP:LIGH:STAT {}".format(self._display)])
+
+    @property
+    def samples(self):
+        return self._samples
+
+    @samples.setter
+    def samples(self, val):
+        # Sample number is per mode.
+
+        # Clip values to upper and lower bounds. DMM likes to crash if set out of bounds
+        if val < 1:
+            val = 1
+        elif val > 1000000:
+            val = 1000000
+
+        self._write(["SENS:COUN {}".format(val)])
```

Comment:
Minor comments that I can see came from the old driver:
- Since there's an instance check in _write: a single string doesn't need to be in a list. 
- f-strings can replace .format